### DISCUSSION TITLE
ParseDCD: Unrecognized DCD format

### DISCUSSION THREAD

**Discussion by jeeberhardt** [prody/ProDy#1882] [Archived Issues]
Hi,

When I merge multiple DCD files with CHARMM and the merge command, ProDy can't recognize it (this condition "if (temp[0] - 4) % 80 != 0" returns False at line 177). But when I use catdcd, it's ok. 

And I don't know why ...

**Comment by cihankayacihan** [comment]
I tried to reproduce this error but I am failed. Can you share a link of your dcd if it is still available?

**Comment by jamesmkrieger** [comment]
I'll look into this now I'm working with CHARMM

**Comment by jamesmkrieger** [comment]
I have managed to reproduce this error now and will see what I can do about fixing it.

**Comment by jamesmkrieger** [comment]
This might help: https://www.charmm.org/ubbthreads/ubbthreads.php?ubb=showflat&Number=4092

**Comment by jamesmkrieger** [comment]
A possible fix could be to read the CHARMM DCDs into VMD and write out new ones. I think those can then be read into ProDy.

**Comment by jamesmkrieger** [comment]
> A possible fix could be to read the CHARMM DCDs into VMD and write out new ones. I think those can then be read into ProDy.

Actually, according to #1534, they can't
